other-docs/guides/upgrading/v5.md

```diff
@@ -0,0 +1,143 @@
+# Upgrading to v5
+
+_If you are migrating from WordPress to Altis, check out the [migrating guide here](../migrating-from-wordpress.md) first._
+
+To upgrade to Altis v5, edit your `composer.json` and change the version constraint for `altis/altis` and any local environment modules to `^5.0.0`:
+
+```json
+{
+	"require": {
+		"altis/altis": "^5.0.0"
+	},
+	"require-dev": {
+		"altis/local-chassis": "^5.0.0",
+		"altis/local-server": "^5.0.0"
+	}
+}
+```
+
+Next remove the `vendor` directory by running `rm -rf vendor` or on Windows `rmdir vendor`. You could also delete the directory using your code editor, Finder or Explorer.
+
+**Note:** due to an issue in how Composer handles installation of `composer-plugin` packages the above step is required to ensure the new version of the required packages are used to manage the process.
+
+Next run `composer update` to complete the upgrade. You should commit the updated `composer.json` and `composer.lock` files.
+
+As Altis v5 requires infrastructure changes, [contact the Cloud team via the Dashboard support](support://new) when deploying v5 to an environment for the first time.
+
+Lastly you will need to update your database tables using the CLI:
+
+- On [Altis Dashboard](https://dashboard.altis-dxp.com/) find the stack you have deployed to and run `core update-db --network` (`wp` is automatically prepended) in the WP CLI tab
+- For Local Chassis run `composer chassis exec -- wp core update-db --network`
+- For Local Server run `composer local-server cli -- core update-db --network`
+
+If you use Local Chassis you will need to also update Chassis and its extensions by running `composer chassis upgrade`.
+
+
+## Breaking Changes
+
+### SEO
+
+The AMP and Facebook Instant Articles plugins have been unbundled from Altis. This allows you to specify the versions of each directly in your Composer requirements. While these are no longer included with Altis, we plan to maintain compatibility with AMP in new features as necessary.
+
+If you're using AMP in your project, after updating your Altis requirements, you will need to additionally require the AMP plugin:
+
+```sh
+$ composer require ampproject/amp-wp@1
+```
+
+Note that while Altis v4 bundles version 1 of the AMP plugin, version 2 is now available with [many major changes](https://github.com/ampproject/amp-wp/releases/tag/2.0.0), so consider updating at the same time.
+
+If you're using Facebook Instant Articles, you will need to require the FBIA plugin. This requires first [adding WordPress Packagist as a repository](docs://getting-started/third-party-plugins#managing-plugins-via-composer), then requiring the plugin from wpackagist:
+
+```sh
+$ composer require wpackagist-plugin/fb-instant-articles
+```
+
```

Comment:
If we add core sitemaps in that can go here too

Reply:
Can handle in follow-up once merged?